cli/src/cli_util.rs

```diff
@@ -916,6 +916,7 @@ impl WorkspaceCommandHelper {
             let (expression, modifier) = self.parse_revset_with_modifier(revision_arg)?;
             let all = match modifier {
                 Some(RevsetModifier::All) => true,
+                Some(RevsetModifier::Prompt) => false,
```

Comment:
nit: I would map `modifier` to `(all, prompt)` here for better compile-time error detection.

Btw, I'm happy to see `let (None | Some(RevsetModifier::All)) = modifier` catches the problem. Perhaps, we'll need a revset_util function that maps `RevsetModifier::Prompt` to parse error.